nin/backend/generate/generate.js

```diff
@@ -53,7 +53,7 @@ const generate = function(projectRoot, type, name, options) {
         if (err) {
           console.error(err);
         } else {
-          console.log('Added ' + shaderLayerName + ' to layers.json');
+          console.log(`-> added ${shaderLayerName} to layers.json`);
```

Comment:
layers.json no longer exists (but this could be fixed in a different pull request).